lua/entities/gmod_time_distortion_generator/init.lua

```diff
@@ -18,59 +18,129 @@ function ENT:Initialize()
 
     local phys = self:GetPhysicsObject()
 
-    if (phys:IsValid()) then
+    self.Radius = 1000
+    self.EntHealth = 100
+    self.EntMaxHealth = 100
+
+    if phys:IsValid() then
         phys:Wake()
     end
+
+    if WireLib then
+        local inNames = {"Activate","Radius"}
+        local inTypes = {"NORMAL","NORMAL"}
+	WireLib.CreateSpecialInputs( self,inNames,inTypes)
```

Comment:
Formatting is a little off here, looks like you've got a tab character instead of spaces and there's a space after the left bracket